src/main/java/com/garbagemule/MobArena/waves/MACreature.java

```diff
@@ -152,6 +142,12 @@ public LivingEntity spawn(Arena arena, World world, Location loc) {
             case "witherskeleton":
                 e.getEquipment().setItemInMainHand(new ItemStack(Material.STONE_SWORD, 1));
                 break;
+            case "piglin":
+                ((Piglin) e).setImmuneToZombification(true);
+            case "piglinbrute":
+                ((PiglinBrute) e).setImmuneToZombification(true);
```

Comment:
This is a pretty small thing, but these two could be merged into one, since the `setImmuneToZombification()` method is declared in the common PiglinAbstract supertype: https://hub.spigotmc.org/javadocs/spigot/org/bukkit/entity/PiglinAbstract.html#setImmuneToZombification(boolean).